crates/aptos-crypto/benches/poseidon_bn254.rs

```diff
@@ -20,11 +20,36 @@ fn bench_group(c: &mut Criterion) {
 
     for n in 1..=16 {
         poseidon_bn254_ark(&mut group, n);
+        poseidon_bn254_neptune(&mut group, n);
```

Comment:
Dumb question, why are these both benchmarked in the same function? Are they still measured separately?